firecrawl.go

```diff
@@ -12,40 +12,60 @@ import (
 	"time"
 )
 
+// RobotsField is a custom type for the robots field in the metadata.
+type RobotsField []string
```

Comment:
i think we need a more robust solution. lots of the metadata fields i'm seeing can be returned as string or a list from testing out the API.

Reply:
Agreed, a bunch of these fields have been returning array of strings instead of just a string for a small set of articles. Seems like this should be fixed on the server side and not client? Would still be great to handle it here gracefully until that's sorted out